web-components/src/[sandbox]/examples/audio-player.ts

```diff
@@ -0,0 +1,9 @@
+import "@/components/audio-player/AudioPlayer";
+import { html } from "lit-element";
+
+export const audioPlayerTemplate = html`
+  <div class="container" style="padding-top: 300px">
```

Comment:
Please avoid using inline styles

Reply:
this is just the sandbox. It is so that the audio player has some room to launch tooltips and playback speed popup